CONTRIBUTING.md

```diff
@@ -1,75 +1,389 @@
+- Set and modify the website title and description.
+- Configure the top navigation bar.
+- Configure the various sidebar menus
 
-Links should be relative and in [this format](https://vuepressbook.com/tutorial/tutorial2.html#linking-to-headers-inside-a-file).
+#### Configuring the top menubar
+Go to `[main/.vuepress/themeConfig/nav.js](/.vuepress/themeConfig/nav.js)` to configure the top
+navigation bar. `/main/.vuepress/config.js`, the overall VuePress configuration file, imports `nav.js`.
 
+Below is an abridged configuration of the top navigation bar showing an array of only two entries,
+Getting Started and Learn More.
+Each entry is an object with three or four properties:
+- `text`: The text shown in the nav bar.
+- `ariaLabel`: Labels this page element. Used to access it if the text is not visible.
+- `link`: Optional. Link to where the browser goes if you click the top menubar item itself, instead of a
+  submenu item. Of the form `link: '/zoe/guide'` where the opening `/` starts the path at `main/`. In this
+  case, no filename was given, so it defaults to `guide`'s `README.md` file. If not present, the menubar
+  entry is not clickable.
+- `items`: Optional. An array of submenu item objects, each of which is a single submenu item of its 
+  parent navbar item. Not present if the item doesn't have a submenu.
+  
+To add an entry to the top menubar, just add a menu item object to the array. We suggest copying a
+similar one, adding the copy, and editing the copy's property values to be those of the new item and
+its submenus. To delete an entry, just remove its object from the menubar item array.
+```js
+module.exports = [
+  {
+    text: 'Getting Started',
+    ariaLabel: 'Getting Started Menu',
+    link: '/getting-started`
+    items: [
+      {
+        // Submenu item
+      },
+      {
+        // Submenu item
+      }
+    ]
+  },     
+  {
+    text: 'Learn More',
+    ariaLabel: 'Learn More Menu',
+    items: [
+      {
+        // Submenu item
+      },      
+      {
+        // Submenu item
+      }
+    ]
+  },  
+]
+```
+#### Configuring the top menubar submenus
+
+A top menubar item without a submenu, such as Glossary, has no
+`items` property and must have a `link` property (otherwise there's
+nothing to click, so it's really not a good navigation menu item).
+```js
+ {
+    text: 'Glossary',
+    ariaLabel: 'Glossary Menu',
+    link: '/glossary/'
+  },
+```
+
+This is an abridged configuration of the `ERTP` entry and its submenu.
+Each submenu item has the same structure and properties as a top menubar item,
+except they do not have an `items` property. Note that for links to files, instead of folders, such as
+the ERTP Introduction, you leave off the file's suffix. So in that case, the
+link value is `'/getting-started/ertp-introduction/'` and not `'/getting-started/ertp-introduction.md'`
+All links must end with a trailing `/`, even if the link is to a file.
```

Comment:
I didn't understand this. I think we should be able to link to an .md no matter what. It's just that in some cases the markdown file is README.md

Reply:
```
All links must end with a trailing `/`, even if the link is to a file.
```
I don't think this is right

Reply:
Doesn't seem to be right. Fixed.

Reply:
I think you're right.